web/styles/settings.css

```diff
@@ -864,21 +864,24 @@ input[type="checkbox"] {
     margin-top: 5px;
 }
 
-.edit_bot_form {
-    font-size: 100%;
-    margin: 0;
-    padding: 0;
+#edit_bot_modal {
+    padding-right: 10px;
```

Comment:
There is no nice way to do this. We should just add `padding-right: 24px` here (since 10px looks very thing too) and removing padding right from header and footer.